components/library/json-c/pkg5

```diff
@@ -6,4 +6,4 @@
         "library/json-c"
     ],
     "name": "json-c"
-}
\ No newline at end of file
+}
```

Comment:
Why is the new 32 bit package not included here?

Reply:
Sorry forgot to also amend this file after the Makefile update - done now.